lib/index.js

```diff
@@ -184,6 +184,8 @@ export default {
 		atom.menu.update();
 
 		Project.onModified(() => {
+			if (!this.toolbar) { return; }
```

Comment:
Linting prefers only 1 statement per line, we can clean this up in the travis PR though